app/views/capture/index2.html.erb

```diff
@@ -1,93 +1,161 @@
   
-      <div class="bs-stepper-header" id="nav-fixed" role="tablist">
+      <div id="testnav" class="bs-stepper-header" role="tablist" style="z-index:100;">
         <%# <!-- your steps here --> %>
         <div class="step" data-target="#landing-page">
-          <button type="button" class="step-trigger" role="tab" aria-controls="landing-page" id="landing-page-trigger">
+          <button type="button" class="step-trigger" role="tab" aria-controls="landing-page" id="homebtn" onclick="homePage()">
             <span class="bs-stepper-label">Home Page</span>
+            <script>
+            function homePage() {
+                if ($("#landing-page-content").hide()) {
+                    $('html').css('background', '#ffffff');
+                    $("#landing-page-content").show();
+
+                    $("div#capture-page-content").hide();
+                    $("div#capture-settings").hide();
+                    $("div#save-page").hide();
+
+                }
+            }
+            </script>
           </button>
         </div>
         <div class="line"></div>
         <div class="step" data-target="#settings">
-          <button type="button" class="step-trigger" role="tab" aria-controls="setting" id="setting-trigger">
+          <button type="button" class="step-trigger" role="tab" aria-controls="setting" id="settingsbtn" onclick="settingsPage()">
             <span class="bs-stepper-label">Settings</span>
+            <script>
+            function settingsPage() {
+                $('html').css('background', '#272727');
+
+                $("#heightIndicator").removeClass("capture-settings-hide");
+                $("#heightIndicator").show();
+                $("#webcam").css('pointer-events', '');
+                $("#webcam").css('opacity', '1');
+
+                $("#capture-page-content").css('height','100vh');
+                $("#capture-page-content").css('width','auto');
+
+                if (($(window).width() < 391 && $(window).height() < 645) || ((window.matchMedia("(orientation: landscape)").matches) && ($(window).height() < 645)))
+                $("#capture-page-content").css('height','auto');
+
+                $("#spectrum-ex").show();
+                $("#capture-option").show();
+
+                $("#landing-page-content").hide();
+                $("div#capture-page-content").show();
+                $("div#capture-settings").hide();
+
+                $("div#save-page").hide();
+
+            }
+            </script>
           </button>
         </div>
         <div class="line"></div>
         <div class="step" data-target="#capture">
-          <button type="button" class="step-trigger" role="tab" aria-controls="capture" id="capture-trigger">
+          <button type="button" class="step-trigger" role="tab" aria-controls="capture" id="capturebtn" onclick="capturePage()">
             <span class="bs-stepper-label">Capture</span>
+            <script>
+            function capturePage() {
```

Comment:
Oof these are rough, huh? So basically the stepper plugin wasn't compatible with the canvas/getusermedia webcam stuff, when run in safari? Is this really the only way forward? It's so much extra code and work! 

Could we at least move these all into one file so as not to mix them with the HTML?

Reply:
The stepper is just stopping the graph so I am rendering all the content first and then hiding it with the scripts instead of using the default stepper behaviour. Sure we could move this to a separate file for a cleaner code.

Reply:
We can also mark them as a temporary step. If we resolve the Safari issue, how much work would it be to revert back, and can we consolidate the code as well as clearly mark it with comments that it's addressing #665 and that upon solving that it could be removed and simplified?

Reply:
Sure, the only thing that would be required in that case would be to simply remove the temporary script and add the pre-defined script section on the page which would like the buttons to the respective divs that would then be rendered as different pages.

Reply:
It may even be helpful to make this particular set of changes all in a single commit. Then we can point at that commit which will include the complete diff of just this steps-fix, and nothing else.